src/Learners/Core/ReceiverOperatingCharacteristic.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Probabilistic.Learners
+{
+    using Microsoft.ML.Probabilistic.Utilities;
+
+    /// <summary>
+    /// Struct which holds a single receiver operating characteristic (ROC)
```

Comment:
Not quite.  The TPR is one characteristic, and the FPR is another.  A better name would be `FalseAndTruePositiveRate`.

Reply:
Renamed to `FalseAndTruePositiveRate` and comment updated.